pages/_app.tsx

```diff
@@ -49,9 +59,26 @@ const App = ({
     <EnvironmentProvider defaultCluster={cluster}>
       <UTCNowProvider>
         <SolanaAccountsProvider>
-          <WalletProvider wallets={getWalletAdapters()} autoConnect>
+          <WalletProvider
+            wallets={[
+              new PhantomWalletAdapter(),
+              new BackpackWalletAdapter(),
+              new SolflareWalletAdapter(),
+              new CoinbaseWalletAdapter(),
+              new BraveWalletAdapter(),
+              new FractalWalletAdapter(),
+              new GlowWalletAdapter(),
+              new LedgerWalletAdapter(),
+              new TorusWalletAdapter(),
+            ]}
```

Comment:
lol okay, this your first problem

These aren't memoized. Every render, you get a new instance of the adapter, destroying its state.

Reply:
I'll send a PR.